src/Memstate.sln

```diff
@@ -24,6 +24,9 @@ EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Memstate.Benchmarks", "Memstate.Benchmarks\Memstate.Benchmarks.csproj", "{DC012469-986E-4953-9E95-BCCF7CD3745A}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Docs", "Docs", "{C51C66C9-3784-4C06-BAC5-EEE4B6684B69}"
+	ProjectSection(SolutionItems) = preProject
```

Comment:
Please don't add item that are not artifacts needed by VS to build. Is this so you can edit in VS?

Reply:
Make sense, done, removed.